wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/petstore/UpdateCartComponent.java

```diff
@@ -26,12 +26,7 @@
 	/**
 	 * The entry field used to enter the quantity of items they wish to purchase.
 	 */
-	private final WTextField amount = new WTextField() {

```

Comment:
The UI in this view is not compliant with our accessibility standards. Could you temporarily replace setAccessibleText with setToolTip and add a TODO to fix the a11y in the UI as a whole.